routes/index.js

```diff
@@ -19,4 +22,24 @@ router.get('/registration', function(req, res) {
   res.render('registration.njk');
 });
 
+router.post('/badgein',jsonParser,function(req,res){
+  if(!req.body)
+    { 
+     return res.sendStatus(400);
+    }
+  var BadgeNumber = req.body.badgeNumber;
+  if(typeof dbAPI.validateUser === undefined)
```

Comment:
I am not sure what this does, why are you checking the type of the DB function and if equal to undefined? (`typeof` returns  a string which represents the name of the type, here you are comparing it to an object `undefined`)